python/llm/test/inference_gpu/test_transformers_api_layernorm.py

```diff
@@ -1,117 +0,0 @@
-#
```

Comment:
Should we change a model for this tests?

Reply:
I'll add new UT after finishing upgrade, in a new way